test/cmockatest.c

```diff
@@ -352,6 +353,62 @@ RTR_TEST_END
 RTR_TEST_START(dirfd)
 RTR_TEST_END
 
+/* netdb functions test */
+RTR_TEST_START(gethostbyname)
+	rtr_gethostbyname("www.google.com");
+RTR_TEST_END
+
+RTR_TEST_START(gethostbyaddr)
+	struct in_addr addr;
+
+	inet_pton(AF_INET, "64.233.189.147", &addr);
+	rtr_gethostbyaddr(&addr, sizeof(addr), AF_INET);
+RTR_TEST_END
+
+RTR_TEST_START(sethostent)
+	rtr_sethostent(1);
+RTR_TEST_END
+
+RTR_TEST_START(endhostent)
+	rtr_endhostent();
+RTR_TEST_END
+
+RTR_TEST_START(gethostent)
+	struct hostent *hent;
+	hent = rtr_gethostent();
+RTR_TEST_END
+
+RTR_TEST_START(gethostbyname2)
+	rtr_gethostbyname2("www.google.com", AF_INET);
+	rtr_gethostbyname2("www.google.com", AF_INET6);
+RTR_TEST_END
+
+RTR_TEST_START(getaddrinfo)
+	struct addrinfo hints, *result;
+
+	memset(&hints, 0, sizeof(struct addrinfo));
+	hints.ai_family = AF_UNSPEC;
+	hints.ai_socktype = SOCK_DGRAM;
+	hints.ai_flags = 0;
+	hints.ai_protocol = 0;
+
+	result = rtr_getaddrinfo("www.google.com", NULL, &hints, &result);
```

Comment:
@efiop It was fault on testing code. Thanks.
But I think it's not need to use assert_int_equal. Because result may be NULL.
Also it should be checked for NULL before free